README.md

```diff
@@ -1,4 +1,23 @@
 # wecount.inclusivedesign.ca
-The source files for the WeCount site
+The source files for the WeCount site.
 
+### How to Run
```

Comment:
@TeddTech These should be H2 elements (`##`) to follow the proper heading order. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
Okay fixing will fix right now